### PR TITLE
OPENEUROPA-1694: Create oe_corporate_blocks component.

### PR DESCRIPTION
**Pull request by nagyad**

### CODE REVIEW

File: grumphp.yml.dist [hunk 1]
```diff
@@ -0,0 +1,15 @@
+imports:
+  - { resource: vendor/openeuropa/code-review/dist/drupal-conventions.yml }
+parameters:
+  tasks.phpcs.ignore_patterns:
+    - vendor/
+    - web/
```
Comment (voidtek) [inline grumphp.yml.dist]: Change to `build`

File: grumphp.yml.dist [hunk 2]
```diff
@@ -0,0 +1,15 @@
...
+  tasks.phpcs.triggered_by:
+    - php
+    - module
+    - inc
+    - theme
+    - install
```
Comment (voidtek) [inline grumphp.yml.dist]: Add `yml` extension

File: .drone.yml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    environment:
+      - MYSQL_ALLOW_EMPTY_PASSWORD=yes
+  selenium:
+    image: selenium/standalone-chrome:3.11
+    environment:
+      - DISPLAY=:99
+      - SE_OPTS=-debug
+
+pipeline:
+  # Download all dependencies.
```
Comment (voidtek) [inline .drone.yml]: The test with lower dependencies is missing.

File: composer.json [hunk 4]
```diff
@@ -0,0 +1,62 @@
...
+            "build/themes/contrib/{$name}": ["type:drupal-theme"]
+        }
+    },
+    "config": {
+        "sort-packages": true,
+        "platform": {
+            "php": "7.1.9"
+        }
+    }
+}
```
Comment (voidtek) [inline composer.json]: End line is missing.

File: runner.yml.dist [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  drupal:site-setup:
+    - { task: "symlink", from: "../../..", to: "${drupal.root}/modules/custom/oe_corporate_blocks" }
+    - { task: "run", command: "drupal:drush-setup" }
+    - { task: "run", command: "drupal:settings-setup" }
+    - { task: "run", command: "setup:phpunit" }
+    - { task: "run", command: "setup:behat" }
+  setup:phpunit:
+    - { task: "process", source: "phpunit.xml.dist", destination: "phpunit.xml" }
+  setup:behat:
+    - { task: "process", source: "behat.yml.dist", destination: "behat.yml" }
```
Comment (voidtek) [inline runner.yml.dist]: End line is missing.

File: .drone.yml (hunk 3 above)
Comment (voidtek) [reply .drone.yml]: Now, the test with higher dependencies is missing. ;)

File: .gitignore [hunk 6]
```diff
@@ -0,0 +1,31 @@
+# Ignore Composer dependencies.
+composer.lock
+/vendor/
+
+# Ignore files generated by IDEs.
+/.idea/
+
+
+# Ignore the web folder except the custom modules and themes
```
[the remark this answers is not in the document]
Comment (voidtek) [reply .gitignore]: not resolved.

File: .drone.yml (hunk 3 above)
Comment (nagyad) [reply .drone.yml]: See line 30

**Pull request merged**